src/wing/CCPACSWingProfile.cpp

```diff
@@ -348,13 +351,16 @@ Handle(Geom2d_TrimmedCurve) CCPACSWingProfile::GetChordLine() const
     return chordLine;
 }
 
+void CCPACSWingProfile::buildPointListAlgo(unique_ptr<CTiglWingProfilePointList>& cache) const
+{
+    cache.reset(new CTiglWingProfilePointList(*this, const_cast<CCPACSPointListXYZ&>(*m_pointList_choice1))); // FIXME(bgruber)
```

Comment:
There is a fixme? What has to be done?

Reply:
the const_cast is the problem. as buildPointListAlgo is const and run concurrently, casting away constness is very dangerous. fortunately, CTiglWingProfilePointList no longer requires a mutable reference, as the point ordering has been moved to CCPACSWingProfile a while ago.

Reply:
Thanks a lot for the clarification :+1: Cool, that you are still around here at the TiGL development :smile: